src/challenges/helpTheBookSeller.js

```diff
@@ -1,28 +1,27 @@
 const stockList = (listOfArt, listOfCat) => {
-  if ((listOfArt.length === 0) | (listOfCat.length === 0)) {
+  let codeTotals = {}
+  let finalResult = ""
+
+  if (listOfArt.length === 0 || listOfCat.length === 0) {
     return ""
   }
-  let quantityOfBooks = {}
-  let result = ""
-  listOfArt.forEach((element) => {
-    if (!(element.split(" ")[0][0] in quantityOfBooks)) {
-      quantityOfBooks[element.split(" ")[0][0]] = parseInt(
-        element.split(" ")[1]
-      )
+  listOfArt.forEach((book) => {
+    let bookCode = book.split(" ")[0][0]
+    let bookNumber = book.split(" ")[1]
```

Comment:
Splitting the string is work you are asking the computer to do, anytime you need to do the same operation back to back, consider using a variable to save the result.